app/assets/javascripts/modules/accordion-with-descriptions.js

```diff
@@ -212,6 +212,7 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       var shouldUpdateHash = true;
 
       this.title = $subsectionElement.find('.js-subsection-title').text();
+      this.element = $subsectionElement;
```

Comment:
What do you think of setting this above and then use `this.element` instead of `$subsectionElement` in this function?